frontend/packages/kubevirt-plugin/src/utils/validations/vm/disk.ts

```diff
@@ -46,6 +46,23 @@ const getEmptyDiskSizeValidation = (): ValidationObject =>
     ValidationErrorType.TrivialError,
   );
 
+const validateBusValue = (
```

Comment:
I thought about it,
but I think it will complicate things as `validateBusValue` returns `ValidationObject` and `validateBus` returns `DiskBusValidationResult`.
additionally, this way I'm preserving the structure of methods at `disk.ts`

What do you think ?

Reply:
Ok let's keep DiskBusValidationResult. Nevertheless, the logic for `ValidationErrorType.Error` and `ValidationErrorType.Warn` could stil be moved to `templateValidations#validateBus`. In that case we could still remove this method and just call `templateValidations.validateBus().getValidationObject()`.